dynamic-demo-plugin/src/components/k8sConsumer/K8sAPIConsumer.tsx

```diff
@@ -10,11 +10,12 @@ import {
   PageSection,
   Title,
 } from "@patternfly/react-core";
-import { k8sCreate, k8sDelete, k8sGet, k8sList, k8sPatch, K8sResourceCommon, k8sUpdate } from '@openshift-console/dynamic-plugin-sdk';
-import { DeploymentModel, mockDeploymetData } from './k8s-data';
+import { getGroupVersionKindForResource, k8sCreate, k8sDelete, k8sGet, k8sList, k8sPatch, K8sResourceCommon, k8sUpdate, useK8sModel } from '@openshift-console/dynamic-plugin-sdk';
+import { mockDeploymetData } from './k8s-data';
 
 const K8sAPIConsumer: React.FC = () => {
   const { t } = useTranslation("plugin__console-demo-plugin");
+  const [k8sModel] = useK8sModel(getGroupVersionKindForResource(mockDeploymetData));
```

Comment:
demo plugin should pass `K8sGroupVersionKind` type to `useK8sModel`

Reply:
`getGroupVersionKindForResource(mockDeploymetData)` returns `K8sGroupVersionKind` only now, so are you suggesting to pass `group~version~kind` directly here?

Reply:
my bad, i thought it returns `group~version~kind` which I wanted to avoid. Current code is ok!

Reply:
Thanks :)